### PR TITLE
Add exhaustive release documentation

### PR DESCRIPTION
**Pull request by amulware**
## What's this?
This adds (almost) exhaustive documentation of how we do releases. A lot of this information previously only existed in my head.

## Why do we want this?
Because I might fall on my head and that would be bad. Also, I do not intend to be handling releases all on my own anymore in the future. And documenting things is neat.

## How is it done?
Markdown, yo.

### Why not another way?
Please.

### Side effects
Developers now can't get out of doing releases themselves.

## Review hints
Read. I expect reviews from several people.

## :squid: Permissions
Me.

### CODE REVIEW

File: docs/release-flow.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+	4. Merge the release branch
+
+Each of these steps, as well as differences between the two platforms are broken down further below.
+
+Note that this is an extension of the steps detailed in our [SuperFlow branching model documentation](https://github.com/toggl/mobile-docs/blob/develop/superflow.md), which also includes a handy summary on [How to make a release](https://github.com/toggl/mobile-docs/blob/develop/superflow.md#how-to-make-a-release).
+
+
+## Release manager
+
+Each release has one developer that is responsible for it. They are the **release manager** for this release. They are responsibly for creating the branch, pull request, changelog and tags. They make sure that the progress and status of the release are communicated, the release is tested well and that all issues found during testing are addressed appropriately. They give the final go-ahead to release to users.
```
Comment (willsb) [inline docs/release-flow.md]: They are responsible* for creating the branch

Comment (willsb) [reply docs/release-flow.md]: Also I'm not sure using the plural applies to `They are the **release manager** for this release`, but I could be wrong

Comment (amulware) [reply docs/release-flow.md]: It's not plural. https://en.wikipedia.org/wiki/Singular_they

Comment (kornelijepetak) [reply docs/release-flow.md]: @amulware don't forget that there were two comments here: `responsibly`

Comment (amulware) [reply docs/release-flow.md]: Hold your horses. Unlike some people, I don't update my PRs overnight. :D

File: docs/release-flow.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+BAD: Updated how tags are stored in database
+BAD: Added more metrics to edit view
+(Users don't have to know about these things.)
+
+#### Use passive voice, group items, and sort by importance
+
+We use passive voice and no full stops when writing our changelogs. Each item should start with `Added`, `Fixed` or `Improved` if possible and are grouped by the same, and in this order, with empty lines between the groups. Each group is ordered from most to least important to our users.
+
+We also often group the many smaller changes and fixes not worth mentioning into a last generic point to indicate that the specific points are just the highlights.
```
[the remark this answers is not in the document]
Comment (amulware) [reply docs/release-flow.md]: I'm not sure which you refer to, but I'm not happy with the sentence and will rephrase it.

**Comment by amulware** [comment]
I fixed the typo @willsb pointed out, rephrased the sentence @kornelijepetak commented on and pushed better numbers to the headers that I already wrote yesterday but forgot to push then.

File: docs/release-flow.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (kornelijepetak) [reply docs/release-flow.md]: I was talking about the `the` in:
`the many smaller`.

**Pull request merged**